.github/linters/.flake8

```diff
@@ -0,0 +1,4 @@
+[flake8]
+# https://black.readthedocs.io/en/stable/compatible_configs.html#flake8
+extend-ignore = E203,W503
+max-line-length = 88
```

Comment:
this seems to contradict the `.flake8` file added to the root dir

Reply:
You are right, I dropped the one in the main directory.

Reply:
So it's breaking the molecule GitHub action: https://github.com/gwarf/ansible-role-umd/actions/runs/3565266345/jobs/5990204325#step:4:38

I will try to find a way to configure it to use the flake conf from the action, even when running outside of GitHub Action, if you think it's OK.

Reply:
So it works now: https://github.com/gwarf/ansible-role-umd/actions